README.md

```diff
@@ -31,10 +31,10 @@ const [value, setValue] = useStorage('mykey');
 
 While Capacitor allows you to write to one API across several platforms, not all features are supported on all platforms. It is encouraged to check if the feature you intend to use is available before using it to avoid any runtime errors.
 
-Each of the hook plugin paths exports an `availableFeatures` object, which contains a list features for that plugin. If the feature is supported for the current platform the app is running on, that feature will be true.:
+Each of the hook plugin paths exports an `availableFeatures` object, which contains a list features for that plugin. If the feature is supported for the current platform the app is running on, that feature will be true:
 
 ```jsx
-const { useStorageItem, availableFeatures } = `@capacitor-community/react-hooks/storage`;
+const { useStorage, availableFeatures } = `@capacitor-community/react-hooks/storage`;
```

Comment:
```suggestion
const { useStorageItem, availableFeatures } = `@capacitor-community/react-hooks/storage`;
```

Reply:
`useStorageItem` is actually correct, the code underneath that uses `useStorage` is wrong. I will file a seperate PR to correct that.